conf/stats.conf.example

```diff
@@ -2,96 +2,105 @@
 type=mysql_query
 statsd_type=gauge
 statsd_ns=source.packetfence.devices.registered_per_role
-mysql_query=select name, count(distinct node.mac) from node_category right join node on node.category_id=node_category.category_id where node.status='reg' group by node_category.category_id;
+mysql_query=select name, count(distinct node.mac) from node_category left join node on node.category_id=node_category.category_id and node.status='reg' group by node_category.category_id
 interval=60s
+randomize=true
 
 [metric 'total number of unregistered devices']
 type=mysql_query
 statsd_type=gauge
 statsd_ns=source.packetfence.devices.unregistered
 mysql_query=select count(distinct mac) from node where status='unreg'
 interval=60s
+randomize=true
 
 [metric 'total online devices']
 type=mysql_query
 statsd_type=gauge
 statsd_ns=source.packetfence.devices.online
 mysql_query=select count(distinct callingstationid) from radacct where acctstoptime is null
 interval=60s
+randomize=true
 
 [metric 'total offline devices']
 type=mysql_query
 statsd_type=gauge
 statsd_ns=source.packetfence.devices.offline
 mysql_query=select count(distinct callingstationid) from radacct where acctstoptime <= NOW()
 interval=60s
+randomize=true
 
 [metric 'number of new registered devices during the past hour']
 type=mysql_query
 statsd_type=gauge
 statsd_ns=source.packetfence.devices.registered_last_hour
 mysql_query=select count(distinct mac) from node where status='reg' and regdate >= NOW() - INTERVAL 1 HOUR
-interval=60s
+interval=1s
 
 [metric 'number of new registered devices during the past day']
 type=mysql_query
 statsd_type=gauge
 statsd_ns=source.packetfence.devices.registered_last_day
 mysql_query=select count(distinct mac) from node where status='reg' and regdate >= NOW() - INTERVAL 1 DAY
-interval=60s
+interval=24s
 
 [metric 'number of new registered devices during the past week']
 type=mysql_query
 statsd_type=gauge
 statsd_ns=source.packetfence.devices.registered_last_week
 mysql_query=select count(distinct mac) from node where status='reg' and regdate >= NOW() - INTERVAL 1 WEEK
-interval=60s
+interval=168s
```

Comment:
I'm not sure if s means hours but otherwise most of these are wrong

Reply:
@julsemaan see my comment above regarding memory sizing, I'm balancing the interval(s) so we have ~3600 samples over the entire duration of the chart's focus.

https://github.com/firehol/netdata/wiki/Memory-Requirements#sizing-memory